### ISSUE TITLE
topchannels stopped responding

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/karmabot#79]
<img width="402" alt="image" src="https://user-images.githubusercontent.com/387927/110655776-a01e2980-81bf-11eb-8cc1-40812341611c.png">

In response to the `@karmabot topchannels` command.

Awesome we had logging:

```
    if not response["ok"]:
        logging.error(f'Error for API call "channels.list": {response["error"]}')
        return "I am truly sorry but something went wrong ;("
```

To find out that:

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/387927/110652267-7f080980-81bc-11eb-95f2-ef9492e0985c.png">

**Comment by pogross** [comment]
So we need to upgrade to the new API #53

**Comment by pogross** [comment]
Will be fixed by #82

**Comment by pogross** [comment]
fixed
